fuse/data/utils/collates.py

```diff
@@ -43,6 +43,7 @@ def __init__(
         keep_keys: Sequence[str] = tuple(),
         raise_error_key_missing: bool = True,
         special_handlers_keys: Optional[Dict[str, Callable]] = None,
+        post_collate_special_handlers_keys: Optional[List[Callable]] = None,
```

Comment:
Why not to use special_handlers_keys instead?
Can you give an example of a use case?

Reply:
a use case is `match_target_key_to_length` from below, we can't use special handlers because it handles one element in the batch dict each time, we need to match one key to another after the special handler has been applied.

Reply:
Why is it a list and not a single callable?

Reply:
to allow multiple use cases, e.g. aligning a set of keys to key A and a different set of keys to key B.